api/src/main/java/org/openmrs/parameter/OrderSearchCriteria.java

```diff
@@ -81,6 +91,49 @@
 	private final boolean excludeDiscontinueOrders;
 
 	/**
+	 * Instead of calling this constructor directly, it is recommended to use {@link OrderSearchCriteriaBuilder}.
```

Comment:
Two questions:
- Is this second constructor even needed, if it isn't even the recommended way to use this?
- If it is, it should not be copying all of this setting logic, but instead first calling the other constructor, and then setting the additional properties.

Reply:
I debated just removing it.  Seemed like it would be safe to remove since people shouldn't be using it directly, but since I'm planning on backporting I figured I'd keep it around to be safe.

But, agreed, I'll change it not to be repetitive... I was thinking that one constructor couldn't call another, but maybe I'm wrong about that...